### PR TITLE
DOCSUP-7046: Edit and translate to Russian

### PR DESCRIPTION
**Pull request by sevirov**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:

- Documentation (edit and translate to Russian)

### CODE REVIEW

File: docs/ru/sql-reference/functions/date-time-functions.md [hunk 1]
```diff
@@ -497,108 +497,98 @@ SELECT now(), date_trunc('hour', now(), 'Europe/Moscow');
...
-```
+**Пример**
 
-Запрос с указанием часового пояса:
+Запрос:
 
-``` sql
-SELECT now('Europe/Moscow');
+```sql
+select date_add(YEAR, 3, toDate('2018-01-01'));
```
Comment (adevyatova) [inline docs/ru/sql-reference/functions/date-time-functions.md:544]:
```suggestion
SELECT date_add(YEAR, 3, toDate('2018-01-01'));
```

File: docs/ru/sql-reference/functions/date-time-functions.md [hunk 2]
```diff
@@ -667,6 +657,167 @@ SELECT date_sub(YEAR, 3, toDate('2018-01-01'));
...
+
+``` text
+┌─now('Europe/Moscow')─┐
+│  2020-10-17 10:42:23 │
+└──────────────────────┘
+```
+
+## today {#today}
+
+Принимает ноль аргументов и возвращает текущую дату на один из моментов выполнения запроса.
```
Comment (adevyatova) [inline docs/ru/sql-reference/functions/date-time-functions.md:813]:
```suggestion
Возвращает текущую дату на момент выполнения запроса. Функция не требует аргументов.
```

File: docs/ru/sql-reference/functions/date-time-functions.md [hunk 3]
```diff
@@ -667,6 +657,167 @@ SELECT date_sub(YEAR, 3, toDate('2018-01-01'));
...
+То же самое, что toDate(now())
+
+## yesterday {#yesterday}
+
+Принимает ноль аргументов и возвращает вчерашнюю дату на один из моментов выполнения запроса.
```
Comment (adevyatova) [inline docs/ru/sql-reference/functions/date-time-functions.md:818]:
```suggestion
Возвращает вчерашнюю дату на момент выполнения запроса.
```

File: docs/ru/sql-reference/functions/date-time-functions.md [hunk 4]
```diff
@@ -667,6 +657,167 @@ SELECT date_sub(YEAR, 3, toDate('2018-01-01'));
...
+Делает то же самое, что today() - 1.
```
Comment (adevyatova) [inline docs/ru/sql-reference/functions/date-time-functions.md:819]:
```suggestion
Делает то же самое, что today() - 1. Функция не требует аргументов.
```

**Comment by sevirov** [comment]
@vdimir Please, review this PR.

**Review by vdimir — APPROVED** [review]

**Pull request merged**